ansible/tasks/build/centos/register-host.yml

```diff
@@ -7,7 +7,7 @@
             ansible_ssh_pass="{{ image_pass }}"
             ansible_connection="ssh"
             ansible_host_key_checking=false
-            ansible_python_interpreter="{{ python_path | default('/usr/bin/python') }}"
+            ansible_python_interpreter="{{ python_path | default('/usr/bin/python3') }}"
```

Comment:
This one cannot be set in general. It would prevent building centos7. Pls define `python_path` as another variable [here](https://github.com/erigones/esdc-factory/blob/ed974c945aafc43b7569cc5f10eee7a2c6b3ac62/ansible/build-contrib-gitlab-ce.yml#L18).